dropwizard-configuration/src/test/resources/factory-test-invalid.json

```diff
@@ -0,0 +1,3 @@
+{
+	"name": "Boop"
+}
```

Comment:
Just a nitpick, could you please add a newline symbol to the end of JSON files? That way they're displayed correctly in terminal windows.